### PR TITLE
Developer REST API Backup endpoints 

### PR DESCRIPTION
**Pull request by yozank**
Added docs for the backup related operations: 

- Create Backup
- Delete 
- Restore
- List 
- Enable Daily Backup
- Disable Daily Backup

### CODE REVIEW

File: devops/developer-api/redis/create_backup.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## Request Parameters
+
+<ParamField body="name" type="string" required>
+  Name of the backup
+</ParamField>
+
+## Response Parameters
+
+<ResponseField name="status" type="string">
```
Comment (fahreddinozcan) [inline devops/developer-api/redis/create_backup.mdx:22]: This endpoint only returns "OK" on successful request. When we say it like this, it looks like it returns {status: "BLABLA"} in the response body. Instead, we can just say "This endpoint returns 'OK' on successfull request".

File: devops/developer-api/redis/delete_backup.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  The ID of the Redis database
+</ParamField>
+
+<ParamField path="backup_id" type="string" required>
+  The ID of the backup to delete
+</ParamField>
+
+## Response Parameters
+
+<ResponseField name="status" type="string">
```
Comment (fahreddinozcan) [inline devops/developer-api/redis/delete_backup.mdx:20]: Same issue with response here

File: devops/developer-api/redis/disable_dailybackup.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## URL Parameters
+
+<ParamField path="id" type="string" required>
+  The ID of the Redis database
+</ParamField>
+
+## Response Parameters
+
+<ResponseField name="status" type="string">
```
Comment (fahreddinozcan) [inline devops/developer-api/redis/disable_dailybackup.mdx:16]: Same issue with this parameter

File: devops/developer-api/redis/restore_backup.mdx [hunk 4]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Restore Backup"
+description: "This endpoint restores a backup for a Redis database."
```
Comment (fahreddinozcan) [inline devops/developer-api/redis/restore_backup.mdx:3]:
```suggestion
description: "This endpoint restores a database data from an existing backup."
```

**Comment by fahreddinozcan** [comment]
Other than that, I feel like redis endpoint list gets a bit crowded, we can collect backup-related files in folder in the same location. They'll just be able to collapse it

File: devops/developer-api/redis/restore_backup.mdx (hunk 4 above)
Comment (joschan21) [reply devops/developer-api/redis/restore_backup.mdx:3]: > 

suggestion: "restores data from an existing backup"

**Review by fahreddinozcan — APPROVED** [review]

**Pull request merged**